tools/dev/msftidy_docs.rb

```diff
@@ -102,7 +102,7 @@ def info(txt, line=0)
   def has_module
     module_filepath = @full_filepath.sub('documentation/','').sub('/exploit/', '/exploits/')
     found = false
-    ['.rb', '.py', '.go'].each do |ext|
+    ['.rb', '.py', '.go', 'jar'].each do |ext|
```

Comment:
Presumably this should use `.jar` rather than `jar` for consistency with the other array items.

```suggestion
    ['.rb', '.py', '.go', '.jar'].each do |ext|
```